File: Ahorn/triggers/SkinSwapTrigger.jl
module SkinModHelperSkinSwapTrigger
using ..Ahorn, Maple

@mapdef Trigger "SkinModHelper/SkinSwapTrigger" SkinSwapTrigger(x::Integer, y::Integer, width::Integer=32, height::Integer=40,
	skinId::String="Default", revertOnLeave::Bool=false)

const placements = Ahorn.PlacementDict(
	"Skin Swap Trigger (Skin Mod Helper)" => Ahorn.EntityPlacement(
		SkinSwapTrigger,
		"rectangle"
	)
)

end

File: Loenn/triggers/SkinSwapTrigger.lua
local skinSwapTrigger = {}

skinSwapTrigger.name = "SkinModHelper/SkinSwapTrigger"
skinSwapTrigger.placements = {
    {
        name = "normal",
        data = {
            width = 32,
            height = 40,
            skinId = "Default",
            revertOnLeave = false,
			playerVariant = true,
			otherselfVariant = true,
			silhouetteVariant = false,
        }
    }
}

return skinSwapTrigger